src/inference/src/dev/core_impl.cpp

```diff
@@ -1077,8 +1077,14 @@ std::shared_ptr<const ov::Model> ov::CoreImpl::apply_auto_batching(const std::sh
         auto pos = deviceName.find_first_of(":");
         if (pos == std::string::npos)
             return model;  // BATCH device is already configured via the config
-        deviceNameWithBatchSize = deviceName.substr(pos + 1);
+
         deviceNameWithoutBatch = ov::DeviceIDParser::get_batch_device(deviceNameWithBatchSize);
+        if (deviceName.find("(") == std::string::npos) {
+            auto supported_properties = ICore::get_property(deviceName.substr(pos + 1), ov::supported_properties, {});
```

Comment:
```suggestion
            auto supported_properties = ICore::get_property(deviceNameWithoutBatch, ov::supported_properties, {});
```
should we use this call?